internal/storage/v2/clickhouse/tracestore/dbmodel/from_dbmodel.go

```diff
@@ -60,97 +59,78 @@ func FromDBModel(dbTrace Trace) ptrace.Traces {
 	return trace
 }
 
-func FromDBResource(r Resource) (pcommon.Resource, error) {
+func convertResource(Span) (pcommon.Resource, error) {
 	resource := ptrace.NewResourceSpans().Resource()
-	resourceAttributes, err := AttributesGroupToMap(r.Attributes)
-	if err != nil {
-		return resource, fmt.Errorf("failed to decode resource attribute: %w", err)
-	}
-	resourceAttributes.CopyTo(resource.Attributes())
+	// TODO: populate attributes
+	// TODO: do we populate the service name from the span?
```

Comment:
@yurishkuro Question for you. My guess was no and that this would already exist in the attributes we store but just wanted to double check.